src/app-layout/visual-refresh-toolbar/skeleton/styles.scss

```diff
@@ -105,7 +112,9 @@
     /* stylelint-disable plugin/no-unsupported-browser-features */
     &:not(:has(> [data-testid])) {
       inline-size: var(#{custom-props.$toolsWidth});
-      border-inline-start: awsui.$border-divider-section-width solid awsui.$color-border-layout;
+      &.tools-open {
```

Comment:
How is this line related to our bugfix?

Reply:
Without this change, the border is displayed also when tools are closed. It becomes visible when `headerVariant="high-contrast"`